setup-r/src/setup-r.ts

```diff
@@ -1,12 +1,29 @@
 import * as core from "@actions/core";
 import { getR } from "./installer";
 import * as path from "path";
+import * as fs from "fs";
+
 
 async function run() {
   try {
     core.debug(`started action`);
-    let version = core.getInput("r-version");
-    core.debug(`got version ${version}`);
+   
+    var version: string
+    version = core.getInput("r-version")
+
+    if (version === "renv") {
+      let renv_lock_path = "./renv.lock";
```

Comment:
Used `.` here as it will be resolved in the calling scope. Better way of doing this?